### DISCUSSION TITLE
Error when creating tailwind css config file

### DISCUSSION THREAD

**Discussion by joshjoshy** [tailwindlabs/tailwindcss#4834] [Help]
I'm trying to setup tailwindcss by using `npm install -D tailwindcss@latest postcss@latest autoprefixer@latest`. After that, I wrote the code `npx tailwindcss init` to create tailwind css config file. But it took to an error like this...

`Object.fromEntries is not a function`
`(node:3912) ExperimentalWarning: The fs.promises API is experimental`

My node version is `v11.0.0`.
NPM version: `6.1.0`

Is I'm doing in the wrong way?

Please try to answer my question!

**Comment by deadcoder0904** [comment]
Try cleaning your cache using `npm cache clean --force`. Also, try deleting `node_modules` & `package-lock.json` & doing a fresh install.

If it still doesn't work then try to clone an existing boilerplate like Next.js + Tailwind :)

**Comment by dance2die** [comment]
Tailwind CSS v.2.x requires Node v12.13.0 or higher.  
https://tailwindcss.com/docs/upgrading-to-v2#upgrade-to-node-js-12-13-or-higher
